src/main/java/com/sda/model/User.java

```diff
@@ -0,0 +1,17 @@
+package com.sda.model;
+
+import jakarta.persistence.Id;
+
+public class User {
+    @Id
+    String username;
+    String password;
+    String name;
+    String surname;
+   Integer age;
+   String email;
+
```

Comment:
Please add access modifiers, , setters, getters, no args constructor and @Entity annotation

Reply:
Please use @Setter, @Getter, @NoArgsConstructor, @AllArgsConstructor, @Builder and @ToString